noxfile.py

```diff
@@ -15,7 +15,7 @@
     sys.exit(1)
 
 package = "ehrapy"
-python_versions = ["3.8"]
+python_versions = ["3.9"]
```

Comment:
Think 3.8 is the standard so far. We should stick to it and you might have to create a 3.8 env for it